src/chapter_3/PDAs.md

```diff
@@ -0,0 +1,315 @@
+# PDAs
+Program derived addresses are addresses with special properties.
+
+Unlike normal addresses, PDAs are not public keys and therefore do not have an associated private key. There are two use cases for PDAs. They provide a mechanism to build hashmap-like structures on-chain and they allow programs to sign instructions.
+
+Working with PDAs is one of the most challenging parts of working with Solana.
```

Comment:
Also a comment along the lines of PDAs being very useful / a super power to further motivate them. I think we really want to encourage the use of PDAs as much as possible since it simplifies the programming model and removes security footguns.